docs/advanced-topics/session-management.md

```diff
@@ -0,0 +1,37 @@
+---
+id: session-management
+title: Session Management
+---
+
+You can use session management to ensure that when a player disconnects, some data is kept and accurately assigned back to that player when they reconnect.
+
+## Linking data to players
+
+To reassign data to the right player when they reconnect, we need to link this data to the player.
+
+The **clientId** generated by Netcode for GameObjects (Netcode) cannot be used, because it generates when a player connects and is disposed of when they disconnect. That ID may then be assigned to a different player if they connect to the server after the first one disconnected.
+
+To properly link data to a specific player, we need an ID that is not tied to the current connection, but persists through time and is unique to that player. Some options include
+* A login system with unique player accounts
+* A Globally Unique Identifier (GUID). For example, a GUID generated via `System.Guid.NewGuid()` and then saved to the `PlayerPrefs` client side.
+
+With this unique identifier, we can map each player's data that is needed when reconnecting, such as the current state of their character. We then ensure this data is up to date and kept after a player disconnects to properly reconnect the player.
+
+You can also decide to clear all data when a session completes or add a timeout to purge the data after a specific amount of time.
+
+## Reconnection
+
+Reconnecting players depends on your game. For example, if we use a [Player Object](../basics/networkobject.md#player-objects), a new `Default Player Prefab`  automatically spawns when any player connects to the game (including when they reconnect). You can then use the player's previously saved session data to update that object so that its state returns to what it was before disconnecting. In those cases, we would need to keep all the important data that we want to restore and map it to the player using our identification system.
+
+In cases where we use don't use the Player Object approach and instead manually attribute client ownership to NetworkObjects, we could keep the objects that a player owns when they disconnect, and set the reconnected player as their new owner. To accomplish this, the only data we would need to keep would be the mapping between those objects and their owning player's identifier, then when a player reconnects we could use this mapping to set them as the new owner.
```

Comment:
Similar to the above comment, do we have a suggestion/best practice on how to set up this mapping and setting them as the new owner?